src/main/io/ledstrip.c

```diff
@@ -484,20 +478,32 @@ static void applyLedFixedLayers()
         }
 
         if (ledGetOverlayBit(ledConfig, LED_OVERLAY_THROTTLE)) {
-            hOffset += scaledAux;
+			if (rcData[THROTTLE] < (PWM_RANGE_MIN + PWM_RANGE_MAX)/2) //if below average
```

Comment:
Thank you for the feedback; I have changed the code so it will work with any of the sticks/aux channelse in the same way as scaledAux. I used a variable called auxInput instead because the scaledAux is scaled to the range of the Hue parameter, and I need to fade all HSV values separately.

I am completely new to Github and don\t know how to get a new file into this conversation for testing et cetera.